torch/_torch_docs.py

```diff
@@ -5298,6 +5298,40 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.heaviside,
+           r"""
+heaviside(input, val, *, out=None) -> Tensor
+
+Computes the Heaviside step function for each element in :attr:`input`.
+
+The Heaviside step function is defined as:
+
+.. math::
+    \text{{heaviside}}(input, val) = \begin{cases}
+        \0, & \text{if input < 0}\\
+        \val, & \text{if input == 0}\\
+        \1, & \text{if input > 0}
+    \end{cases}
+""" + r"""
+Args:
+    {input}
+    val (Float point Tensor or value): The value of the element where is 0 in input. 
+                                       If input.shape != val.shape, they must be broadcastable to a common shape 
+                                       (which becomes the shape of the output).
+
+Keyword arguments:
+    {out}
+
+Example::
+
+    >>> a = torch.tensor([-1.5, 0, 2.0])
+    >>> torch.heaviside(a, 0.5)
+    tensor([0, 0.5, 1.0])
+    >>> torch.heaviside(a, 1)
```

Comment:
Probably want a distinct value that, like -2 or 3.

Reply:
(Or you could just kill this example. The first one does a good job.)

Reply:
Addressed!